staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_test.go

```diff
@@ -180,42 +180,42 @@ func TestList(t *testing.T) {
 	featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ConsistentListFromCache, false)
 	ctx, cacher, server, terminate := testSetupWithEtcdServer(t)
 	t.Cleanup(terminate)
-	storagetesting.RunTestList(ctx, t, cacher, compactStorage(cacher, server.V3Client.Client), true)
+	storagetesting.RunTestList(ctx, t, cacher, compactStore(server.V3Client.Client), true)
```

Comment:
hmm - those tests are weird.

Because they don't use compact to do compaction, but (even as it's stated in the comment in some of those tests):
```
// Use compact to increase etcd global revision without changes to any resources
```

Can we instead change those tests so that they don't take `compaction Compaction` function as an argument, but rather introduce a new type like `increaseRV IncreaseRVFunc` or sth so that it's clear what we're doing?